### ISSUE TITLE
Spawning arrows with /aa

### ISSUE THREAD

**Issue by netherfoam** [alkarinv/BattleArena#40]
Bet you're sick of my bug reports, but... If you do /aa addspawn arrow 64 0 30 or whatever numbers you like, when the arrows are spawned they are entity arrows and not item arrows, so nobody can pick them up or use them.

You may want to reorder the searching through the items/mob names or let us use explicit types somehow?

Sort of related:
Spawning potions doesn't work, I'm guessing because you're saving the data value instead of the durability value? Always save the durability value because the durability value is a short (or whatever..) compared to a data value which is a byte.  Potion damage values go above 255, which means they need to be stored in a short, which means that fetching the data value of a potion will result in bad numbers, whereas fetching its durability will result in the correct ones.

TLDR;
Spawning potions doesnt work after a reboot.

**Comment by alkarinv** [comment]
Nah this is good, without people like you these bugs would just linger and piss other people off.  It also makes me wonder why they often don't report anything!  

So spawning , spawning enchanted items, spawning potions.  Plus the changing classes after they've already selected and changed items.

**Comment by netherfoam** [comment]
Yeah. I don't know if you tried to put this in the last version, but the bytes and shorts aren't working yet on potions.

Also, everyone is getting white wool heads instead of the ones they should :/

**Comment by alkarinv** [comment]
white wool... and potions... 

Alright let's see what's going on.

**Comment by alkarinv** [comment]
you always bring me the best bugs ;)

**Comment by alkarinv** [comment]
so white wool is fixed, I put in a very obvious bug in 3.6.9.4 that I didnt see. Potions should be fixed as well.

**Comment by netherfoam** [comment]
On the last version you just uploaded (yay, subscriptions! thanks for telling me that), it seems when you tell a player what potions they get, it's being doubled (the number of messages) for each player playing Eg
player1 gets Speed:1 regen:2 message
player2 gets Speed:1 regen:2 Speed:1 regen:2 message
player3 gets Speed:1 regen:2 Speed:1 regen:2 Speed:1 regen:2 message

Screenshot here: https://dl.dropbox.com/u/72835893/potions.png

**Comment by alkarinv** [comment]
Wow, I must be doing something strange... looking into it

**Comment by alkarinv** [comment]
Ok, so that problem is fixed now version 3.6.9.7+ has the fix.  Turns out it's been in there for ages... how do these things not get reported!!!  You can download that 3.6.9.7 from the github downloads, but I'm going to hold off releasing it to bukkit until I can get some other features/fixes in.
